test_rules_scala.sh

```diff
@@ -27,6 +27,7 @@ $runner bazel test //test/... --extra_toolchains="//test_expect_failure/plus_one
 . "${test_dir}"/test_build_event_protocol.sh
 . "${test_dir}"/test_compilation.sh
 . "${test_dir}"/test_deps.sh
+. "${test_dir}"/test_diagnostics_reporter.sh
```

Comment:
I mean that do an 
if (bazel --version >= "no version") then test_disagnostics_reporter.sh
With a comment that says that once X feature with link will be released then "no version" will be replaced to that bazel version. Wdyt?

Reply:
Ah yeah that's what I considered as well but I understood you wanted something else.